cloud/google/gce.py

```diff
@@ -141,7 +147,7 @@
 
 requirements:
     - "python >= 2.6"
-    - "apache-libcloud >= 0.13.3"
+    - "apache-libcloud >= 0.17.0"
```

Comment:
this is not a change we do lightly, we normally prefer to have version detection code in the modules that will act according to which version is available. Otherwise this forces existing installations to upgrade libcloud when updating Ansible and that is not something we want to do.